docs/projects/ansible/collection/centurion/playbooks/teams.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Teams
+description: No Fuss Computings Companion Ansible Collection Centurion teams Playbook.
+date: 2024-08-18
+template: project.html
+about: https://gitlab.com/nofusscomputing/projects/ansible/collections/kubernetes
+---
+
+The teams playbook has been created for the purpose of creating standardised teams with a defined set of permissions. With this playbook it is possible to create every team within an organisation and define set teams for newly created team. The playbook can also be used to patch a team with updated permissions or notes.
+
+The teams playbook includes the [AWX Feature](../../../playbooks/awx.md) to import the playbook as a job template in AWX / Ansible Automation Platform.
+
+The following job template will be created:
+
+*  **Centurion/Access/Teams** Creation and patching of teams and permissions
+
+On import to AWX / Ansible Automation Platform a credential type will also be created, `Playbook/teams/centurion` that can be used to supply the required secrets and Centurion host.
+
+!!! info Centurion ERP API
+    The playbook is designed to use the Centurion ERP API to fetch, create and patch teams and permissions.
```

Comment:
the playbook doesn't fetch from centurion. The inventory (centurion) plugin as part of this collection obtains what is required from centurion.

Reply:
Adjusted